CMake/cdat_modules/vtk_external.cmake

```diff
@@ -73,6 +73,12 @@ if(CDAT_BUILD_OFFSCREEN)
   )
 endif()
 
+if(CDAT_BUILD_VTKWEB)
```

Comment:
@doutriaux1 we don't want to expose vtkweb. Web option in the future will enable more than just vtkWeb.